tests/unit/test_lower_bound_checker.py

```diff
@@ -34,9 +41,19 @@
 BAD_PACKAGE = "invalid-package"
 
 
+def skip_test_if_not_installed(package_name: str):
```

Comment:
Style note: this could be a decorator for each test function, and pass the package name to the test function, so we only hard-code it once.

Not a blocker by any means.